### ISSUE TITLE
Auto fix silly mistakes

### ISSUE THREAD

**Issue by williamFalcon** [PyTorchLightning/pytorch-lightning#1551]
1. Detect when user has shuffle=True for validation and test data
2. Detect when a layer does not use bias=False right before a batchnorm.

Anyone interested in adding these checks?

**Comment by DavidAdamczyk** [comment]
Yes. I would like to add these checks.

**Comment by geekswaroop** [comment]
@williamFalcon  I would like to take up this issue. Could you point out where exactly the shuffling problem arises?

**Comment by awaelchli** [comment]
I think I know a cool way to do 2)
Mind if I self assign myself?
